<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimpleMsgBox</class>
 <widget class="QWidget" name="SimpleMsgBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>281</width>
    <height>127</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SimpleMsgBox</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/*主界面样式表*/
QWidget
{
	border:none;
	color:white;
	font-family:微软雅黑;
	font-size:14px;
	outline:0 px;
	background:transparent;
}

#topWgt{border-image:url(image/top.png) 2;border-width:2;}
#closeBtn{border-image:url(image/close.png) 2;border-width:2;}
#bgWgt{border-image:url(image/bg.png) 2;border-width:2;}
#iconLbl{border-image:url(image/notice.png) 2;border-width:2;}
/*1-按钮开始*/
QPushButton
{
	border:none;
	border-image:url(F:/data/images/button/button-n.png) 3;
	border-width:3;
}
QPushButton:hover
{
	border-image:url(F:/data/images/button/button-h.png) 3;
	border-width:3;
}
QPushButton:pressed
{
	border-image:url(F:/data/images/button/button-s.png) 3;
	border-width:3;
}
/*1-按钮结束*/</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="topWgt" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>32</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>32</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="titleLbl">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>32</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>32</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>248</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="closeBtn">
        <property name="minimumSize">
         <size>
          <width>22</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>22</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="bgWgt" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>4</number>
      </property>
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>9</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>24</width>
            <height>17</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="iconLbl">
          <property name="minimumSize">
           <size>
            <width>36</width>
            <height>36</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>36</width>
            <height>36</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="textLbl">
          <property name="minimumSize">
           <size>
            <width>66</width>
            <height>26</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>25</width>
            <height>17</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>24</number>
        </property>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>27</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="okBtn">
          <property name="minimumSize">
           <size>
            <width>66</width>
            <height>26</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>66</width>
            <height>26</height>
           </size>
          </property>
          <property name="text">
           <string>确定</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="cancelBtn">
          <property name="minimumSize">
           <size>
            <width>66</width>
            <height>26</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>66</width>
            <height>26</height>
           </size>
          </property>
          <property name="text">
           <string>取消</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>28</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
